extension/screenshooter/src/main/java/org/arquillian/extension/recorder/screenshooter/browser/impl/BrowserScreenshooter.java

```diff
@@ -145,18 +146,18 @@ public Screenshot takeScreenshot(File screenshotToTake, ScreenshotType type) {
         return screenshoot;
     }
 
-    private WebDriver getTakingScreenshotsBrowser() {
-
-        GrapheneContext context = null;
+    private WebDriver getTakingScreenshotsBrowser(File screenshotToTake) {
+        WebDriver result = null;
 
         try {
-            context = GrapheneContext.getContextFor(Default.class);
+            GrapheneContext context = GrapheneContext.getContextFor(Default.class);
+            result = ((GrapheneProxyInstance) context.getWebDriver(TakesScreenshot.class)).unwrap();
```

Comment:
Ok, now i see that you catch the IllegalStateException. Although i don't like catching RuntimeExceptions like these, this works. Thanks for your changes!